app/assets/javascripts/bastion/layouts/nutupane.html

```diff
@@ -19,7 +19,7 @@ <h2 class="col-sm-2">
     </div>
 
 
-    <div class="col-sm-3 nutupane-info">
+    <div class="col-sm-3 nutupane-info" ng-show="table.rows.length > 0">
```

Comment:
Because it looked strange when there were no rows but you're right that it will disappear when you search for something that doesn't exist which is not what was intended, will fix.